app/src/main/java/com/example/rraya/bookshelf/Adapter/BookAdapter.java

```diff
@@ -44,7 +44,7 @@ public long getItemId(int i) {
     public View getView(int i, View view, ViewGroup viewGroup) {
         View item = LayoutInflater.from(viewGroup.getContext()).inflate(R.layout.book_item,viewGroup, false);
         TextView textView = (TextView) item.findViewById(R.id.book_title);
-        textView.setText(bookList.get(i).getTitle());
+        textView.setText(bookList.get(i).getVolumeInfo().getTitle());
```

Comment:
@Mathreyu consider to display the book thumbnail on list items and implement [view holder patter](https://developer.android.com/training/improving-layouts/smooth-scrolling.html#ViewHolder) and the reason of behind this patter :) and then move this to a recycler view to know the differences between list views and recycler views.